packages/main/src/Avatar.js

```diff
@@ -190,6 +210,7 @@ class Avatar extends UI5Element {
 	}
 
 	static async onDefine() {
+		await fetchI18nBundle("@ui5/webcomponents");
```

Comment:
Switch to
await Promise.all([
fetchI18nBundle("@ui5/webcomponents"),
Icon.define(),
])